dashboard/src2/pages/NewAppTrial.vue

```diff
@@ -272,6 +272,18 @@ export default {
 				name: this.productId
 			};
 		},
+		site() {
+			return {
+				type: 'document',
+				method: 'get_product_redirect_url',
```

Comment:
don't need method for document resource 
https://frappeui.com/story/docs-resources-document-resource-story-js#options

Reply:
okay will update it.